### PR TITLE
Updtaed README file

### PR DESCRIPTION
**Pull request by Shraddhak22**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package main
```
Comment (sbose78) [inline main.go:1]: Why do we have this file here?

**Review by sbose78 — CHANGES_REQUESTED** [review]
Thank you for your contribution, have a few questions!

File: main.go (hunk 1 above)
Comment (Shraddhak22) [reply main.go:1]: A readme file for potential users to know what this project is all about

Comment (sbose78) [reply main.go:1]: But this is a main.go

**Comment by sbose78** [comment]
What's up with this PR ?

**Comment by Shraddhak22** [comment]
I am closing this PR as the requested changes available in PR,
https://github.com/cli-playground/kodo/pull/24 (It has updated README file)

**Pull request closed without merging**